src/it/scala-2.12/org/apache/james/gatling/JamesServer.scala

```diff
@@ -15,9 +18,10 @@ object JamesServer {
   class RunningServer(container: GenericContainer[_]) {
     lazy val mappedJmapPort: Integer = container.getMappedPort(imapPort)
     lazy val mappedWebadminPort: Integer = container.getMappedPort(webadminPort)
-    private lazy val administration = new JamesWebAdministration(new URL(s"http://localhost:$webadminPort"))
 
-    def addUser(user: User): Unit = administration.addUser(user)
+    private def administration = new JamesWebAdministration(new URL(s"http://localhost:$mappedWebadminPort"))
+
+    def addUser(user: User): Unit = Await.result(administration.addUser(user), 30 seconds)
```

Comment:
we should define a constant somewhere to define how much we want to wait during setup phase for async things.